skins/oasis/modules/LicenseController.class.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class LicenseController extends WikiaController {
+
+	public function executeIndex() {
+		global $wgRightsText, $wgRightsUrl;
+
+		$license = Html::element( 'a', [ 'href' => $wgRightsUrl ], $wgRightsText );
+		$description = wfMessage('license-description')->inContentLanguage()->rawParams($license)->escaped();
```

Comment:
we could move html building into the template, and then the method could be empty and the template could be rendered via `renderPartial`

Reply:
k, I'll do that, thanks!